extensions/whox.md

```diff
@@ -30,21 +30,26 @@ The second argument contains a percent character followed by a list of fields
 requested by the client. Each field is represented by a letter. The standard
 fields are:
 
-- 't': return the `<token>` specified by the client
-- 'c': return an arbitrary channel the client is joined to
-- 'u': return the username
-- 'i': return the IP address
-- 'h': return the hostname
-- 's': return the server name
-- 'n': return the nickname
-- 'f': return the `WHO` flags (away, server operator, etc)
-- 'l': return the number of seconds the user has been idle for
-- 'a': return the account name
-- 'r': return the realname
+- `t`: return the `<token>` specified by the client
+- `c`: return an arbitrary channel the client is joined to
+- `u`: return the username
+- `i`: return the IP address
+- `h`: return the hostname
+- `s`: return the server name
+- `n`: return the nickname
+- `f`: return the `WHO` flags (away, server operator, etc)
+- `d`: return the hop count (distance)
+- `l`: return the number of seconds the user has been idle for
+- `a`: return the account name
+- `o`: return the channel op level
+- `r`: return the realname
 
 Servers MAY support additional non-standard fields. Servers MUST NOT rely on
 the ordering of the fields.
 
+Clients SHOULD avoid requesting the hop count (`d`) and the channel op
```

Comment:
I would say "avoid relying on the value of" instead. Requesting them can be useful to increase entropy when sending automated tokenised WHOX requests to distinguish them from manual client sent ones.

Reply:
Hm, can you elaborate? I'm not sure I understand why it can be desirable for a client to request these.

Reply:
It's not a particularly strong reason tbh, and I don't think it needs to be mentioned in the spec, but the more I can make an automated WHO request differ from a potential user requested one the better.

More importantly, clients could have any number of valid reasons for requesting this information that we can't predict so I don't know if it warrants a stern telling off in the spec for doing so.